OM-Demo/AdListViewController.swift

```diff
@@ -70,22 +70,59 @@ class AdListViewController: UITableViewController {
     var adUnits: [AdUnit] = [.HTMLDisplay, .nativeVideo, .nativeDisplay, .nativeAudio]
 
     override func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+        return 2
```

Comment:
If we consider reverting this to a single-section table view, we can also delete `numberOfSections(in:)` implementation entirely